staging/operator-lifecycle-manager/pkg/controller/registry/reconciler/grpc.go

```diff
@@ -408,7 +419,7 @@ func imageID(pod *corev1.Pod) string {
 
 func (c *GrpcRegistryReconciler) removePods(pods []*corev1.Pod, namespace string) error {
 	for _, p := range pods {
-		err := c.OpClient.KubernetesInterface().CoreV1().Pods(namespace).Delete(context.TODO(), p.GetName(), *metav1.NewDeleteOptions(1))
+		err := c.OpClient.KubernetesInterface().CoreV1().Pods(namespace).Delete(context.TODO(), p.GetName(), metav1.DeleteOptions{})
```

Comment:
I think before we merge this, I'd like to see whether we want a package-level constant for the default value (e.g. `metav1.DeleteOptions{}`) that has a comment that links to both of those BZs. Does that sound reasonable?

Reply:
Oh yeah, that's a good point - I'm not 100% sure whether we'd want a comment to that in upstream OLM (I see the argument for both sides), but that's a discussion we can have once we determine whether these fixes close out the attached BZ anyways.